Templates/MonoGame.Templates.VisualStudio/MonoGame.Templates.VSExtension.csproj

```diff
@@ -0,0 +1,83 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Name of the csproj is not the same as the name of the folder, is this intentional?

Reply:
Is it okay to you if I rename everything "MonoGame.Templates.VSExtension" down to the output vsix?

I concede that the initial proposal was too verbose, but I feel like it might be too confusing to have a "VisualStudio" folder within a "Templates" folder not containing actual templates but an extension.

Reply:
Yup, I'm fine with renaming everything to `MonoGame.Templates.VSExtension`